__tests__/app.test.js

```diff
@@ -81,3 +80,37 @@ describe("GET /api/reviews/:review_id", () => {
             });
     });
 });
+
+describe("GET /api/reviews", () => {
+    test("200: Responds with an array of review objects", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                expect(reviews).toHaveLength(13);
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        designer: expect.any(String),
+                        review_img_url: expect.any(String),
+                        votes: expect.any(Number),
+                        category: expect.any(String),
+                        owner: expect.any(String),
+                        comment_count: expect.any(Number),
+                    });
+                });
+            });
+    });
+    test("200: Responds with array of objects sorted by date in descending order", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                const reviewsCopy = [...reviews];
+                expect(reviewsCopy).toBeSortedBy('created_at', { descending: true });
```

Comment:
You don't need a copy here, as there's no risk of mutation, so feel free to just pass in reviews here.